x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/metric.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { css } from '@emotion/react';
+import { EuiFlexItem, EuiBadge, EuiText, UseEuiTheme } from '@elastic/eui';
+
+type Type = 'cpuTime' | 'events' | 'eventsEmitted' | 'eventMillis';
+
+const metricStyle = (type: Type) => (theme: UseEuiTheme) => {
+  let width: string;
+
+  switch (type) {
+    case 'cpuTime':
+      width = theme.euiTheme.size.xxl;
+      break;
+    case 'events':
+    case 'eventsEmitted':
+      width = `calc(${theme.euiTheme.size.xxl} * 4)`;
+      break;
+    case 'eventMillis':
+      width = `calc(${theme.euiTheme.size.xxl} * 2)`;
+      break;
+    default:
+      width = 'auto';
+  }
+
+  return css`
+    text-align: right;
+    width: ${width};
+
+    @media (min-width: ${theme.euiTheme.breakpoint.m}) {
+      text-align: left;
+      padding-left: ${theme.euiTheme.size.xl};
+    }
+  `;
+};
+
+const metricFlexItemStyle = (theme: UseEuiTheme) => css`
+  @media (min-width: ${theme.euiTheme.breakpoint.m}) {
+    margin-bottom: ${theme.euiTheme.size.xs} !important;
```

Comment:
@mgadewoll I absolutely have no idea whether it is needed or not. I prefer to keep just in case.

Reply:
Generally speaking since we touch/change things we should test them, meaning we probably could just have a look if removing it changes anything.

But I'm ok keeping it 👍

Reply:
`!important` was introduced in https://github.com/elastic/kibana/pull/68726, maybe @andreadelrio can share some insights.

Reply:
@consulthys could you provide the prerequisites and the test path? I'll try removing the flag and see what happens.

Reply:
@weronikaolejniczak You need to resize the screen below the breakpoint (or use a different device with a small screen, I believe it was 768px) and then that rule kicks in.